spec/features/idv/in_person_spec.rb

```diff
@@ -121,6 +121,39 @@
     complete_all_in_person_proofing_steps
   end
 
+  it 'allows the user to go back to document capture from prepare step', allow_browser_log: true do
+    sign_in_and_2fa_user
+    begin_in_person_proofing
+
+    # location page
+    expect(page).to have_content(t('in_person_proofing.headings.location'))
+    bethesda_location = page.find_all('.location-collection-item')[1]
+    bethesda_location.click_button(t('in_person_proofing.body.location.location_button'))
+
+    # prepare page
+    expect(page).to have_content(t('in_person_proofing.headings.prepare'))
+    click_button t('forms.buttons.back')
+
+    expect(page).to have_content(t('in_person_proofing.headings.location'))
+    click_button t('forms.buttons.back')
+
+    # Note: This is specifically for failed barcodes. Other cases may use
+    #      "idv.failure.button.warning" instead.
+    expect(page).to have_button(t('doc_auth.buttons.add_new_photos'))
```

Comment:
When testing this feature, I notice that both the description of the error and the "Verify your ID in person" links are absent when clicking "Back". Could we add an assertion for that, and if we want to split that out as a separate ticket, can we still include them, but comment them out and point to the corresponding Jira ticket?

![image](https://user-images.githubusercontent.com/1779930/183425418-bcef0eef-0d01-48bb-aea7-881f3c747fdb.png)

Reply:
Looks like there's a ticket for the bug at LG-6874